mods/ra/sequences/infantry.yaml

```diff
@@ -64,7 +64,8 @@ e1:
 		Start: 324
 		Length: 18
 	die6: electro
-		Length: *
+		Frames: 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13
+		Length: 29
```

Comment:
Wouldn't it be better to keep `Length: *` here?

Reply:
I wanted a test case for both definitions.

Reply:
The testcase has shown it works, now lets please change this back to match our normal conventions :)